example-clients/ipload.c

```diff
@@ -199,6 +234,9 @@ main (int argc, char *argv[])
 		}
 	}
 
+	if (autoclose_opt)
+		jack_deactivate(client);
+
```

Comment:
That is ok, the main point is the line after the empty line, there is this function call (line 240) that has a totally different indentation. (At least when i look at the source on github)
Maybe you could correct the indent of that line, not adding braces should be OK if that indent is corrected in my eyes.